.gitmodules

```diff
@@ -1187,6 +1187,9 @@
 [submodule "vendor/grammars/typst-grammar"]
 	path = vendor/grammars/typst-grammar
 	url = https://github.com/michidk/typst-grammar.git
+[submodule "vendor/grammars/vento"]
+	path = vendor/grammars/vento
+	url = https://github.com/oscarotero/vento.git
```

Comment:
There should only be one entry per language in this file and it should be for the grammar as added by the `add-grammar` script. The script also sorts the file. This is not the grammar repo so current being in this file.

Reply:
My bad. I run `script/add-grammar` twice. The first one passing this repository which gives me an error. So I run again with the vscode repository which contains the grammar. I guess this line was added in the first run. Will remove it.